test/support/utils.rb

```diff
@@ -45,6 +43,8 @@ def debug_code(program, boot_options: '-r debug/start', remote: true, &block)
 
       debug_on_local boot_options
 
+      check_line_num!(program)
```

Comment:
When writing a new test case, verifying the test case is valid is more important than adding the line numbers.
So the framework should wait for the test to pass before asking for line numbers.